eth-bytecode-db/eth-bytecode-db/README.md

```diff
@@ -0,0 +1,47 @@
+# Ethereum bytecode database
+
+
+## Development
+
+> all commands should be executed in `eth-bytecode-db/eth-bytecode-db` dir
+
+### Postgres
+
+```bash
+docker run -p 5432:5432 --name eth-bytecode-postgres -e POSTGRES_PASSWORD=admin -d postgres
```

Comment:
When trying to run a migration against database in the docker container, it returns an error with the following message - "database \"eth-bytecode-db\" does not exist". 

I suggest to either add a description of how the database can be created inside docker container before running the migrations, or to create a database if the one does not already exiist.